docs/patrol/alt-native-setup.mdx

```diff
@@ -0,0 +1,298 @@
+        testInstrumentationRunnerArguments clearPackageData: "true"
+      ```
+
+      5. Add this section to the `android` section:
+
+      ```groovy
+        testOptions {
+          execution "ANDROIDX_TEST_ORCHESTRATOR"
+        }
+      ```
+
+      6. Add this line to `dependencies` section:
+
+      ```groovy
+        androidTestUtil "androidx.test:orchestrator:1.4.2"
+      ```
+
+#### iOS setup
+
+      1. Open `ios/Runner.xcworkspace` in Xcode. 
+      
+      2. Create a test target if you do not already have one via `File > New > Target...`
+         and select `UI Testing Bundle`. Change the `Product Name` to `RunnerUITests`. Make
+         sure `Target to be Tested` is set to `Runner` and language is set to `Objective-C`.
+         Select `Finish`.
+
+      3. 2 files are created: `RunnerUITests.m` and `RunnerUITestsLaunchTests.m`.
+         Delete `RunnerUITestsLaunchTests.m` **through Xcode**.
+
+      4. Make sure that the **iOS Deployment Target** of `RunnerUITests` within the
+         **Build Settings** section is the same as `Runner`.
+         The minimum supported **iOS Deployment Target** is `13.0`.
+
+      ![Xcode iOS deployment target](/assets/ios_deployment_target.png)
+
+      ![Xcode iOS deployment target 2](/assets/ios_deployment_target_2.png)
+
+      5. Replace contents of `RunnerUITests.m` file with the following:
+
+      ```objective-c title="ios/RunnerUITests/RunnerUITests.m"
+      @import XCTest;
+      @import patrol;
+      @import ObjectiveC.runtime;
+
+      PATROL_INTEGRATION_TEST_IOS_RUNNER(RunnerUITests)
+      ```
+
+      Add the newly created target to `ios/Podfile` by embedding in the existing
+      `Runner` target.
+
+      ```ruby title="ios/Podfile"
+      target 'Runner' do
+        # Do not change existing lines.
+        ...
+
+        target 'RunnerUITests' do
+          inherit! :complete
+        end
+      end
+      ```
+
+      6. Create an empty file `integration_test/example_test.dart`. From the command line, run:
+
+      ```
+      $ flutter build ios --config-only integration_test/example_test.dart
+      ```
+
+      7. Go to your `ios` directory and run:
+
+      ```
+      $ pod install --repo-update
+      ```
+
+      8. Open your Xcode project and make sure every target has correct base
+         Build Configurations:
+
+      ```
+      $ open Runner.xcworkspace
+      ```
+
+      ![Xcode config setup](/assets/ios_runner_configs.png)
+
+      9. Add 2 new "Run Script Phase" Build Phases to the RunnerUITests target:
+
+      ![Xcode config setup](/assets/ios_runner_build_phases.png)
+
+      10. Paste this code into the first `xcode_backend build` Build Phase:
+
+      ```
+      /bin/sh "$FLUTTER_ROOT/packages/flutter_tools/bin/xcode_backend.sh" build
+      ```
+
+      11. Paste this code into the second `xcode_backend embed_and_thin` Build Phase:
+
+      ```
+      /bin/sh "$FLUTTER_ROOT/packages/flutter_tools/bin/xcode_backend.sh" embed_and_thin
+      ```
```

Comment:
I think this is the most problematic part of the iOS setup, maybe it's good to paste here screenshots with pasted code?

Reply:
So instead of the picture before, you'd like a picture after?

Reply:
Exaclty, I want the picture with pasted code